app/views/toolbar.js

```diff
@@ -41,6 +41,16 @@ const Toolbar = Mn.LayoutView.extend({
         this.listenTo(application.channel, 'search:done', () => {
             this.ui.search.removeClass('loading');
         });
+
+        this.listenTo(application.channel, 'drawer:toggle', () => {
+            let drawer = this.$el.parent().get(0);
+            let isExpanded = drawer.getAttribute('aria-expanded') == 'true';
+            drawer.setAttribute('aria-expanded', !isExpanded);
+        });
+        this.listenTo(application.appState, 'change:currentPlaylist', () => {
```

Comment:
Here too: it's just reacting to the `appState` change, why not putting it directly to the view's parent?